### PR TITLE
Epns juicebox goodkarma

### PR DESCRIPTION
**Pull request by Rashmi-278**
Add EPNS, Juicebox and GoodkarmaDAO

### CODE REVIEW

**Comment by cryptosabago** [comment]
@Rashmi-278 I have a qn about: protocols/graphprotocol.eth/gip.png is it the correct logo? Should it be named `logo.png`?

File: protocols/GoodKarma/index.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "cname": "example",
+  "description": "A short description of dao/protocol",
+  "path": "example",
+  "isEnabled": true,
+  "discourseForum": {
```
Comment (cryptosabago) [inline protocols/GoodKarma/index.json]: I think that these should be deleted if the ingo cannot be found.

File: protocols/GoodKarma/index.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "cname": "example",
```
Comment (cryptosabago) [inline protocols/GoodKarma/index.json]: Please fill in cname, path, and description

File: protocols/GoodKarma/overview.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Overview of protocol and governance process.
```
Comment (cryptosabago) [inline protocols/GoodKarma/overview.md]: Please fill this in.

**Comment by Rashmi-278** [comment]
Hey, it's not logo it's an image for overview page. Logo is different.

**Review by cryptosabago — APPROVED** [review]
Code looks good, just one qn regarding an icon that does not look right. Small nit: you might want to drop some files from some of the commits.

**Comment by cryptosabago** [comment]
> Hey, it's not logo it's an image for overview page. Logo is different.

Do we care if logos are `png` or `jpeg`? 👀  @ yungapesquad logo

**Pull request merged**